Source/Core/Core/IOS/USB/Emulated/Skylander.cpp

```diff
@@ -748,8 +749,9 @@ bool SkylanderPortal::CreateSkylander(const std::string& file_path, u16 sky_id,
     memcpy(&file_data[(index * 0x40) + 0x36], &other_blocks, sizeof(other_blocks));
   }
   // Set the skylander info
-  u16 sky_info = (sky_id | sky_var) + 1;
-  memcpy(&file_data[0], &sky_info, sizeof(sky_info));
+  u8 nuid[4];
+  Common::Random::Generate(&nuid, 4);
+  memcpy(&file_data[0], &nuid, sizeof(nuid));
```

Comment:
Why go through a temporary instead of just Generate(&file_data[0], ...) ?

Reply:
Good question. Didn’t think of that. Will fix

Reply:
Fixed in latest commit